### DISCUSSION TITLE
Asking for help for Crud operation

### DISCUSSION THREAD

**Discussion by bunthondev** [loopbackio/loopback-next#8215] [Q&A]
Hi everyone,
I am new with loopback, i have 2 models, user and asset model
Do loopback provide crud that can insert user and it's assets in one request?

**Comment by achrinza** [comment]
Yes, it's possible to call multiple repository functions within a single HTTP request. However it's not possible perform in a single database insert unless if done through custom SQL queries with `.execute()`
